Coupled-Cluster/spin_free_CC/polar.py

```diff
@@ -0,0 +1,84 @@
+# A simple Psi 4 script to compute CCSD from a RHF reference
+# Scipy and numpy python modules are required
+#
+# Algorithms were taken directly from Daniel Crawford's programming website:
+# http://sirius.chem.vt.edu/wiki/doku.php?id=crawdad:programming
+# Special thanks to Lori Burns for integral help
+#
+# Created by: Daniel G. A. Smith
+# Date: 7/29/14
+# License: GPL v3.0
+#
+
+import time
+import numpy as np
+from helper_CCENERGY import *
```

Comment:
Since Python places no limits on the ratio of classes to files (since each file is regarded as a reusable module, see [here](http://docs.python-guide.org/en/latest/writing/structure/#modules) for more details) I think the code would be cleaner if each of these classes were placed in the `helper_CC.py` module, and imported as, e.g.,
```
from helper_CC import helper_CCENERGY
```
That way, (a) the `ndot()` function would not need to be duplicated in every class file, and (b) the overall directory structure would be cleaner, with only one helper module present.